### ISSUE TITLE
Latest xsnap won't install off npm

### ISSUE THREAD

**Issue by danfinlay** [Agoric/Agoric#6195]
## Describe the bug
Failing to install xsnap via npm.

## To Reproduce

1. Create a new project and initialize it. `mkdir test && cd test && npm init`
2. Enter `npm i @agoric/xsnap`
3. See error

```
npm i @agoric/xsnap -S
npm ERR! code 1
npm ERR! git dep preparation failed
npm ERR! command /root/.nvm/versions/node/v16.17.0/bin/node /root/.nvm/versions/node/v16.17.0/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/root/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
npm ERR! > rollup@2.70.1-endo.1 prepare
npm ERR! > husky install && npm run build
npm ERR!
npm ERR!
npm ERR! > rollup@2.70.1-endo.1 build
npm ERR! > shx rm -rf dist && node scripts/update-git-commit.js && rollup --config rollup.config.ts --configPlugin typescript && shx cp src/rollup/types.d.ts dist/rollup.d.ts && shx chmod a+x dist/bin/rollup
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN deprecated acorn-dynamic-import@4.0.0: This is probably built in to whatever tool you're using. If you still need it... idk
npm ERR! fatal: not a git repository (or any of the parent directories): .git
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! node:internal/modules/cjs/loader:959
npm ERR!   throw err;
npm ERR!   ^
npm ERR!
npm ERR! Error: Cannot find module '/root/.npm/_cacache/tmp/git-cloneuAeHWP/scripts/update-git-commit.js'
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:804:27)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
npm ERR!     at node:internal/main/run_main_module:17:47 {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: []
npm ERR! }
npm ERR! npm ERR! code 1
npm ERR! npm ERR! path /root/.npm/_cacache/tmp/git-cloneuAeHWP
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command sh /tmp/prepare-ca4990ee.sh
npm ERR!
npm ERR! npm ERR! A complete log of this run can be found in:
npm ERR! npm ERR!     /root/.npm/_logs/2022-09-14T17_29_54_387Z-debug-0.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-09-14T17_29_45_126Z-debug-0.log
```

## Platform Environment
Reproduced on both Mac OS X and Ubuntu, using both node 16 and node 18 via nvm.

**Comment by mhofman** [comment]
We probably should figure out how to publish this properly. Either we include pre-built binaries, or we need to make the build step compatible with the regular node build system.

cc @dckc

**Comment by dckc** [comment]
Did this ever work? Is this a regression?

I filed #3857 a long time ago with the goal of moving our getting started docs from `git clone; yarn && yarn build` to `npm install` norms (where `yarn install` is in the ballpark too). It's closed even though we're not there; I think there's another issue to represent the rest of the work, but I don't see it just now.

Re xsnap in particular, the current approach where `yarn build` builds from source and git submodules and all that dates back to Aug 2020 https://github.com/Agoric/agoric-sdk/issues/1299#issuecomment-668963339 . I think we considered pre-built binaries... we didn't decide to spend the effort to do that, but yes, it would be nice. I don't think we considered the node build system.

**Comment by mhofman** [comment]
This should have been fixed as a side effect of #7883, but we still need to publish a `latest` release with this fix. The `dev` tag should work.

**Comment by turadg** [comment]
Confirmed fixed
